docs/zkapps/how-to-write-a-zkapp.mdx

```diff
@@ -282,6 +279,28 @@ Under the hood, every `@method` defines a zk-SNARK circuit. From the cryptograph
 
 :::
 
+<!-- TODO: create at least a basic section about zk circuits below this subsection which contains (among other things) a more thourough explanation of what this tip hints at -->
+
+:::tip
+
+You will find that inside a `@nethod`, things sometimes behave a little differently. For example, the following code can't be used in a method where `x: Field` is an input parameter:
```

Comment:
Would suggest we reword this section to lead with the most important thing, and then follow with the explanation. It will be shorter and they'll know what the entire section is about. But let's skip that today, to save time.